framework/src/mesh/MooseMesh.C

```diff
@@ -500,21 +500,23 @@ MooseMesh::cacheChangedLists()
 }
 
 ConstElemPointerRange *
-MooseMesh::refinedElementRange()
+MooseMesh::refinedElementRange() const
```

Comment:
You made this function `const`, but the thing it returns is not `const`, so someone can modify it.  Is this function really `const` then?